### PR TITLE
[RM Core Appendix A] Initial structure

### PR DESCRIPTION
**Pull request by petorre**
Resolving issue: https://github.com/cntt-n/CNTT/issues/294

### CODE REVIEW

File: doc/ref_model/chapters/appendix-a.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+* [A.3 VNF Maturity Levels](#A.3)
+* [A.4 Links](#A.3)
 
-<a name="10.1"></a>
-## 10.1 Heading 1.
+<a name="A.1"></a>
+## A.1 Goals
+This Appendix has two goals:
+1. Provide guidance to VNF or more generally Application vendors on how to consume CNTT Reference Model and Architectures
+2. Provide usable definitions of maturity levens for VNF software architecture between Physical-to-Virtual migration and “Cloud Native”.
```
Comment (deebhatia) [inline doc/ref_model/chapters/appendix-a.md]: `levens` -> `levels`

**Comment by rabi-abdel** [comment]
@petorre can you please get the issue #294 assigned to you by @markshostak?

**Comment by rabi-abdel** [comment]
looks good @petorre

**Comment by rabi-abdel** [comment]
@deebhatia can you please approve this so it can be merged?

**Comment by deebhatia** [comment]
> @deebhatia can you please approve this so it can be merged?

Done.

**Pull request merged**